views/legislation-page.js

```diff
@@ -192,10 +192,10 @@ const measureListRow = (s) => {
             <h3><a href="${measureUrl}">${s.title}</a></h3>
             ${s.introduced_at ? html`
             <div class="is-size-7 has-text-grey">
-              <span class="has-text-weight-bold">${s.short_id.replace(/^[^-]+-(\D+)(\d+)/, '$1 $2').toUpperCase()}</span> &mdash;
-              ${s.sponsor_first_name
-                ? html`Introduced by&nbsp;<a href=${`/${s.sponsor_username}`}>${s.sponsor_first_name} ${s.sponsor_last_name}</a>&nbsp;on ${(new Date(s.introduced_at)).toLocaleDateString()}`
```

Comment:
Why remove this `Introduced by`?

Reply:
The previous line was a sentence (Introduced by...on...). That doesn't flow as well with the policy area included

Open to other suggestions, but this seemed cleanest

HR 1957 — John Lewis • Taxation • 3/28/2019

HR 1957 — Introduced by John Lewis on 3/28/2019 — Taxation